pymc/variational/updates.py

```diff
@@ -534,9 +534,7 @@ def adagrad(loss_or_grads=None, params=None, learning_rate=1.0, epsilon=1e-6):
 
     for param, grad in zip(params, grads):
         value = param.get_value(borrow=True)
-        accu = aesara.shared(
-            np.zeros(value.shape, dtype=value.dtype), broadcastable=param.broadcastable
-        )
+        accu = aesara.shared(np.zeros(value.shape, dtype=value.dtype), shape=param.broadcastable)
```

Comment:
```suggestion
        accu = aesara.shared(np.zeros(value.shape, dtype=value.dtype), shape=param.shape)
```

?

Reply:
Yes, but we don't _need_ to make those kinds of changes (yet), because `shape` is backward compatible with `broadcastable` (i.e. it takes a list of bools that yields the same results).